jdaviz/core/registries.py

```diff
@@ -75,10 +77,12 @@ def add(self, name, cls, label=None):
             in the first parameter.
         label : str, optional
             The label displayed in the tooltip when hovering over the tray tab.
+        overwrite : bool, optional
+            Whether to overwrite an existing entry with the same ``label``.
```

Comment:
Maybe add one more sentence to explain what would happen to the old thing that was overwritten?

Reply:
suggestions for wording?  It is replacing it in the registry, so calls to access the item will retrieve the new one instead.

Reply:
Is the old one going to be destroyed somewhere or just sitting in purgatory? I am not familiar with this machinery.

Reply:
This doesn't do anything to destroy it, no, it just replaces the reference in the registry.

Reply:
Should we destroy it?

Reply:
I don't see any reason to, its generally just a class and shouldn't be instantiated yet